message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -20,6 +20,7 @@
    [cmr.message-queue.config :as config]
    [cmr.message-queue.services.queue :as queue])
   (:import
+   (cmr.message_queue.test ExitException)
```

Comment:
I don't think we should include test namespaces in our src ones. I think it is fine to move ExitException to src.